client/rpcserver/handlers.go

```diff
@@ -840,6 +868,19 @@ var helpMsgs = map[string]helpMsg{
       See https://github.com/satoshilabs/slips/blob/master/slip-0044.md`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(walletLockedStr, "[coin symbol]") + `"`,
+	},
+	toggleWalletStatusRoute: {
+		pwArgsShort: "appPass",
+		argsShort:   `assetID disable`,
+		cmdSummary:  `Disable or enable an existing wallet.`,
+		pwArgsLong: `Password Args:
+    appPass (string): The DEX client password.`,
+		argsLong: `Args:
+    assetID (int): The asset's BIP-44 registered coin index. e.g. 42 for DCR.
+      See https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+	disable (bool): The wallet's status. e.g To disable a wallet set to "true", to enable set to "false"`,
```

Comment:
Could you move disable to the left more? I think it's just four spaces and not tab.
Looks like this now.
```
$ ./dexcctl --simnet help togglewalletstatus
togglewalletstatus assetID disable

Disable or enable an existing wallet.

Args:
    assetID (int): The asset's BIP-44 registered coin index. e.g. 42 for DCR.
      See https://github.com/satoshilabs/slips/blob/master/slip-0044.md
        disable (bool): The wallet's status. e.g To disable a wallet set to "true", to enable set to "false"

Returns:
    string: The message "[coin symbol] wallet has been [wallet status]"
```